wgsl/index.bs

```diff
@@ -7131,7 +7131,6 @@ can be rewritten as
 `{ let p = &(`|e1|`); *p = *p + (`|e2|`); }`
 </blockquote>
 where the identifier `p` is chosen to be different from all other identifiers in the program.
-</p>
 
 <p algorithm="translation compound assignment vector component">When
```

Comment:
Same should be done for this `<p>`. I bet the `</div>` below was originally meant to match with this `<p>`.

Reply:
I tried to apply the suggestion with nested `div`. Does it look better now?